codegen/src/TypeMapper.scala

```diff
@@ -5,6 +5,43 @@ import scala.meta._
 import scala.meta.dialects.Scala33
 import besom.codegen.metaschema._
 
+import scala.annotation.unused
+
+case class TypeToken private (provider: String, module: String, `type`: String, @unused dummy: Boolean = false) {
```

Comment:
```suggestion
case class TypeToken (provider: String, module: String, `type`: String) {
```
and remove the 3-parameter overload of `apply`

Reply:
the overridden apply has `enforceNonEmptyModule(module),`, so I'm afraid it might be necessary

Reply:
but thank you for noticing the dummy is not necessary after all :)